ports/seal/portfile.cmake

```
@@ -3,49 +3,39 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO microsoft/SEAL
-    REF 0b058d99b7f18a00e5ebb2b80caee593804b0500
-    SHA512 eed4e4360e32c8506a8f84b9bcfd2c8debf0013ee181f37fb34c6b94e62bbde1993ee400a3f589fe2378b598dd4fdb2e8532b706691a9348e464cd1a202ef23c
+    REF d045f1beff96dff0fccc7fa0c5acb1493a65338c
+    SHA512 9b5d3c4342608d8e3d9826d3b52cbefc1c21eb0094d0cae4add8bb0960f931e9080f248eb8ad8385fc0a08e2a1da10020185148ffd2ef02e7a4fac879e27aa69
     HEAD_REF main
 )
 
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
-    ms-gsl SEAL_USE_MSGSL
-    zlib SEAL_USE_ZLIB
-    zstd SEAL_USE_ZSTD
-
+        ms-gsl SEAL_USE_MSGSL
+        zlib SEAL_USE_ZLIB
+        zstd SEAL_USE_ZSTD
```

I see no feature macro added to respect the `hexl` feature; perhaps you meant to add something like
```suggestion
        zstd SEAL_USE_ZSTD
    INVERTED_FEATURES
        hexl CMAKE_DISABLE_FIND_PACKAGE_HEXL
```

---

Thanks for catching this. I've added `SEAL_USE_INTEL_HEXL` to the latest commits.

---

@WeiDaiWD,INVERTED_FEATURES No need?

---

Nope, it is controlled by a single SEAL_USE_XXX argument.